docs/source/integrations/building-integrations.md

```diff
@@ -18,9 +18,10 @@ Server in their web framework of choice.
 The primary responsibility of an Apollo Server integration is to translate
 requests and responses between a web framework's native format to the format used by `ApolloServer`. This article conceptually covers how to build an integration, using the [Express integration](https://github.com/apollographql/apollo-server/blob/version-4/packages/server/src/express4/index.ts) (i.e.,`expressMiddleware`) as an example.
 
-
 > For more examples, see these Apollo Server 4 [integrations demos for Fastify and Lambda](https://github.com/apollographql/server-v4-integration-demos/tree/main/packages).
 
+If you are building a serverless integration, we recommend prepending your function name with the word `start` (e.g., `startAndCreateLambdaHandler(server)`). This is a helpful naming convention for your integration's users and nicely aligns with the names of other Apollo Server integrations such as `startStandaloneServer(server)`. 
```

Comment:
Maybe be even more explicit like "this lets us maintain a standard where every server should call exactly one function or method with the word `start` in its name" or something?

Also @trevor-scheer does the current version of the Lambda example do this?

Reply:
Nope, nor did we do it in DX's lambda implementation

https://github.com/apollographql/server-v4-integration-demos/blob/main/packages/lambda/src/index.ts

Reply:
Does changing the "start this server" function name of each of those integrations something we want to do? I agree that it would be great to set a naming convention, but we probably shouldn't do so until all existing integrations (that we have a say in) align with that naming convention.

Reply:
Yeah, on it

Reply:
https://github.com/apollographql/server-v4-integration-demos/pull/9

Reply:
Cool, how does this tweaked version feel @glasser :

> If you are building a serverless integration, we **strongly recommend** prepending your function name with the word `start` (e.g., `startAndCreateLambdaHandler(server)`). This naming convention helps maintain Apollo Server's standard that every server uses a function or method whose name contains the word `start` (such as `startStandaloneServer(server)`.

Reply:
looks good although we settled on startServerAndCreateLambdaHandler

Reply:
Oh my bad, thank you so much!

Reply:
we changed it after the discussion above :)